_prose.yml

```diff
@@ -1,6 +1,11 @@
 prose:
-  rooturl: 'source'
-  ignore: '.keep'
+  rooturl:
+  ignore:
+    - '.keep'
+    - /lib
+    - /spec
+    - /templates
+    - /middleman-blog-similar
```

Comment:
Good idea to ignore these files that are irrelevant for most day-to-day site changes